backend/copr_backend/rpm_builds.py

```diff
@@ -133,6 +137,18 @@ def predicate(x):
         super().__init__(predicate, limit, name="arch_{}".format(architecture))
 
 
+class ArchitectureUserWorkerLimit(GroupWorkerLimit):
+    """
+    Limit number of machines of specific architecture we give to a single
+    Copr owner (user or group).
+    """
+    def __init__(self, architecture, limit):
+        def hasher(x):
```

Comment:
This probably doesn't have to be nested if we make it `@staticmethod`

Reply:
that can be lambda, going to change...